packages/fxa-auth-server/test/remote/account_create_tests.js

```diff
@@ -29,6 +36,9 @@ describe('remote account create', function () {
     mockStripeHelper.hasActiveSubscription = async () => Promise.resolve(false);
     mockStripeHelper.removeCustomer = async () => Promise.resolve();
 
+    Container.set(PlaySubscriptions, {});
+    Container.set(AppStoreSubscriptions, {});
```

Comment:
@bbangert , would love your thoughts on this.

I had to add this to a bunch of the remote tests to get them to pass.

For some reason, now that I'm throwing an error if there's no `StripeHelper` in these classes, that error is getting thrown in these tests, despite `TestServer.start` starting the auth server via `key_server.js`, which should set `StripeHelper` on the `Container`.

Reply:
I have a guess, but it's probably not terribly re-assuring.The test runner apparently can start a test before a previous one has completed its tear-down, this is why running one in isolation may work while the entire test suite could have outstanding promises that haven't quite finished. It's possible there's a test option that may help with this, but the easier solution is the hack you've done.

Reply:
I filed https://mozilla-hub.atlassian.net/browse/FXA-5134 as a maintenance spike to look into this a bit more.